x-pack/platform/plugins/shared/maps/public/react_embeddable/types.ts

```diff
@@ -72,10 +70,6 @@ export type MapApi = DefaultEmbeddableApi<MapSerializedState> &
 
 export const isMapApi = (api: unknown): api is MapApi => {
   return Boolean(
-    api &&
-      apiIsOfType(api, 'map') &&
-      typeof (api as MapApi).getLayerList === 'function' &&
-      apiPublishesTitle(api) &&
-      apiPublishesUnifiedSearch(api)
+    api && (api as HasType)?.type === 'map' && typeof (api as MapApi).getLayerList === 'function'
```

Comment:
Did something change recently that allowed us to be less explicit? We don't need to check if `apiPublishesTitle` and `apiPublishesUnifiedSearch` anymore?

Reply:
Makes sense. Thanks. 👍